takoyaki/src/main/java/frc/robot/Constants.java

```diff
@@ -29,7 +29,7 @@
   }
   
   public static final class kHopper {
-    public static final int MOTOR_ID = -1;
+    public static final int MOTOR_ID = 10;
```

Comment:
```suggestion
    public static final int MOTOR_CAN_ID = 10;
```
Also reconsider renaming MOTOR to actual name of motor/controller.